Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -221,9 +221,9 @@ - (void)stopAnimating {
         _isAnimating = NO;
     }
 
-    [_progressRing setIsActive:YES];
+    [_progressRing setIsActive:NO];
```

Comment:
Is it possible that calling stopAnimating displays the spinner without the animation. Wouldn't your change hide it now?